scripts/bump_versions.sh

```diff
@@ -50,7 +50,12 @@ find . -name "Cargo.toml" | while read -r cargo_file; do
 
   # If we changed anything, overwrite the file
   if $changed; then
-    printf "%s\n" "${content[@]}" > "$cargo_file"
+    # Fix: Use a loop to write each line separately to avoid formatting issues.
```

Comment:
Would you mind removing this line? I don't think including explanations of fixes in the code is a good idea (but the context you kept below is great).